docs/narratives/index.rst

```diff
@@ -0,0 +1,14 @@
+======================================
+Communicating Results Using Narratives
+======================================
+
+This page shouldn't be rendered & should jump to the first TOC entry instead. TODO.
+
+.. toctree::
+   :hidden:
+   :maxdepth: 0
+   :titlesonly:
+
+   introduction
+   how-to-write
```

Comment:
@jameshadfield these pages 404 (e.g. https://docs.nextstrain.org/projects/auspice/en/migrate-docs/narratives/how-to-write) now that we exclude them from the build. Should I go through and replace references to them with their URLs in the main RTD since probably users have not saved URLs like the one I mention here?